README.md

```diff
@@ -20,7 +20,9 @@ running the app for the first time
 | `SECRET_KEY`   | The secret encryption key used to encrypt sessions. Make sure that it's hard to guess |
 | `TIMESYNC_URL` | The URL of the TimeSync server that this app will use     |
 
-For a full list of configuration options, check out the documentation
+Note: If you are only intending to run the app in a development environment,
```

Comment:
"in a development environment" could be changed to "in a development environment with Docker" for extra clarity.